### PR TITLE
Disable Fomantic's CSS tooltips

### PR DESCRIPTION
**Pull request by silverwind**
[CSS-only tooltips](https://fomantic-ui.com/modules/popup.html#tooltip) suffer various issues with positioning and there was only one single instance of them in the templates. Replace that instance with a regular popup and exclude these `data-tooltip` styles from the Fomantic build.

### CODE REVIEW

**Review by 6543 — APPROVED** [review]

File: templates/repo/settings/options.tmpl [hunk 1]
```diff
@@ -170,7 +170,7 @@
 							{{$address := MirrorRemoteAddress .}}
 							<td>{{$address.Address}}</td>
 							<td>{{$.i18n.Tr "repo.settings.mirror_settings.direction.push"}}</td>
-							<td>{{if .LastUpdateUnix}}{{.LastUpdateUnix.AsTime}}{{else}}{{$.i18n.Tr "never"}}{{end}} {{if .LastError}}<div class="ui red label" data-tooltip="{{.LastError}}">{{$.i18n.Tr "error"}}</div>{{end}}</td>
+							<td>{{if .LastUpdateUnix}}{{.LastUpdateUnix.AsTime}}{{else}}{{$.i18n.Tr "never"}}{{end}} {{if .LastError}}<div class="ui red label poping up" data-content="{{.LastError}}">{{$.i18n.Tr "error"}}</div>{{end}}</td>
```
Comment (delvh) [inline templates/repo/settings/options.tmpl:173]: Will the style class `poping` ever be renamed to `popping`?
It simply sounds like a typo…
Or do I misunderstand that style class?

Comment (silverwind) [reply templates/repo/settings/options.tmpl:173]: Actually I think this `poping up` stuff might be a idiosyncrasy we introduced and I think it's actually a complete noop and can be removed, Fomantic only cares about `data-content`. I'm just following the tradition in adding it 😆

Comment (silverwind) [reply templates/repo/settings/options.tmpl:173]: Actually, there seems to be some JS functionality linked to `poping`, we can just rename it.

**Comment by silverwind** [comment]
@delvh included the rename `poping` to `popping` here in the second commit, I ran:

```bash
perl -p -i -e 's#poping#popping#g' templates/**/* web_src/js/index.js

**Comment by delvh** [comment]
If we are already at it: Is there a reason why these two are two separate classes?
Because as far as I can see, they could be combined into one style class `pop-up`.
Then everyone knows without a doubt what they are meant to do.
Doing this makes everything more readable, smaller, and less error-prone in my opinion.
Unless of course that style class is already used somewhere else…
Even though doing this might help reduce bugs in the end as those things are also already intended to popup and we then have a more unified code base…

**Comment by silverwind** [comment]
I think that class can probably be removed entirely. I'll probably move the renaming commit to another PR as it's not directly related.

**Comment by delvh** [comment]
I am fine with both.
I prefer removing it entirely if possible because less code means fewer bugs…

**Review by delvh — APPROVED** [review]

**Comment by silverwind** [comment]
Backed out of the renaming change here, will follow up with a separate PR later.

**Review by axifive — APPROVED** [review]

**Review by 6543 — APPROVED** [review]

**Comment by tomaswarynyca** [comment]
:rocket:

**Pull request merged**